fem/bilinearform.hpp

```diff
@@ -525,6 +531,9 @@ class BilinearForm : public Matrix
    /// Sets diagonal policy used upon construction of the linear system
    void SetDiagonalPolicy(DiagonalPolicy policy);
 
+   /// Indicate that integrators are not owned by the BilinearForm
+   void SetExternBFS(int e = 1) { extern_bfs = e; };
```

Comment:
I will suggest a move descriptive name and dropping the parameter, e.g. `void UseExternalIntegrators()'.

Reply:
+1 for a more descriptive name